tests/unit/s2n_server_new_session_ticket_test.c

```diff
@@ -558,34 +556,33 @@ int main(int argc, char **argv)
                 uint8_t *ticket = s2n_stuffer_raw_read(&session_stuffer, ticket_size);
                 EXPECT_NOT_NULL(ticket);
                 EXPECT_BYTEARRAY_EQUAL(ticket, test_ticket, ticket_size);
-            }
+            };
 
-            /* Check the serialized ticket_age_add is what was in the arbitrary nst message*/ 
+            /* Check the serialized ticket_age_add is what was in the arbitrary nst message*/
```

Comment:
Maybe this isn't common enough to be important, but can clang format add spaces after /* and before */ if they don't exist?

Reply:
Clang-format doesn't mess with block comments like this (although it can do exactly this for line comments). This might be a good candidate for simple checks. Writing a regex to find this isn't that hard.